packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -96,7 +96,6 @@ const defaultUser = {
   launchdarklyTrackingSubgroup: 'dev',
   launchdarklyTrackingId: '111',
   launchdarklyTrackingTeam: undefined,
-  launchdarklyTrackingTenant: 'gcp-eu',
```

Comment:
Why did you remove this?

Reply:
It is deprecated and was causing the test to fail. I guess it has been replaced by launchdarklyTrackingCloudEnvironment.